README.md

```diff
@@ -68,6 +68,8 @@ Some useful command line parameters are:
 - `--server.port=9090` changes the http port (from default 8080) to 9090.
 - `--bom-base.scan-licenses=false` disables the source code license scanner,
   reducing the machine load during development and testing.
+- `--bom-base.use-clearly-defined=false` disables the clearly-defined lookup,
+  for debug purposes when developing harvesters.
```

Comment:
I think it is just a switch to disable ClearlyDefined for whatever reason, and not just for debugging. (e.g. if ClearlyDefined is discontinued by Microsoft.)

(My suggestion is to drop this explicit reason.)

Reply:
Removed reason